app/Models/User.php

```diff
@@ -1022,6 +1022,11 @@ public function maxFriends()
         return $this->isSupporter() ? config('osu.user.max_friends_supporter') : config('osu.user.max_friends');
     }
 
+    public function beatmapsetDownloadAllowance()
+    {
+        return $this->isSupporter() ? config('osu.beatmapset.download_limit') : config('osu.beatmapset.download_limit_supporter');
```

Comment:
i think this is back to front

Reply:
and this is why we have code reviews!... I need caffeine >.>